src/index.js

```diff
@@ -19,19 +19,24 @@ export default function(text, chunkSize) {
   assertIsValidChunkSize(chunkSize);
 
   const chunks = [];
-  while (text.length > chunkSize) {
-    const splitAt = text.lastIndexOf(' ', chunkSize);
+  let characters = [...text];
```

Comment:
I just realised that it matters where this is supported: node 6+

If it would be Array.from() it would be node 4+